cpp/src/ray/util/process_helper.cc

```diff
@@ -60,10 +60,38 @@ static std::string GetNodeIpAddress(const std::string &address = "8.8.8.8:53") {
   }
 }
 
-void ProcessHelper::StartRayNode(int redis_port, std::string redis_password) {
+std::string FormatResourcesArg(const std::unordered_map<std::string, int> &resources) {
+  std::ostringstream oss;
+  oss << "{";
+  for (auto iter = resources.begin(); iter != resources.end();) {
+    oss << "\"" << iter->first << "\":" << iter->second;
```

Comment:
Can we use `nlohmann/json` to generate the JSON? Concating raw strings is unsafe.